files/environments/jupyterlab.yaml

```diff
@@ -13,9 +13,17 @@ dependencies:
   # dask
   - dask-gateway
   - dask-labextension>=5.0.0
-  # dashboard
+  # panel
+  - jupyter-panel-proxy
+  # required to launch jupterlab from jupyterhub
+  - jupyterhub
+  # dashboards (versions specified to narrow conda solve space)
   - cdsdashboards-singleuser
-  - panel
```

Comment:
Add panel as dependency

Reply:
panel is a dependency below (I added a version to narrow conda solve space)

Reply:
Whoops sorry about that!